### DISCUSSION TITLE
Release compatible with last LTS Quarkus Version

### DISCUSSION THREAD

**Discussion by amauryLiagre** [quarkiverse/quarkus-bucket4j#146] [General]
Hi, 

I just wonder if a new release of bucket4j compatible with the last Quarkus LTS version (3.15.1) is planned ? And in that case, can you give an estimated date please ? 

I'm asking because for our project, we want to update the quarkus version to 3.15.1 (actually 3.8.5). And as this library seems to use old resteasy reactive extensions, the build failed with this error : 

"Failed to resolve artifact io.quarkus:quarkus-resteasy-reactive-spi-deployment:jar:3.15.1"

That's a consequency of the migration "[RESTEasy Reactive extensions renamed to Quarkus REST](https://github.com/quarkusio/quarkus/wiki/Migration-Guide-3.9#resteasy-reactive-extensions-renamed-to-quarkus-rest-gear-white_check_mark)" from 3.9.X Quarkus Version.

Thank you for this very useful plugin.

**Comment by antoinechamot** [comment]
I noticed that a merge was made into the main branch to upgrade the version. I guess release take some time, it seems there is a PR to improve the release workflow

**Comment by rmanibus** [comment]
hey guys, I just broke out a new release (1.0.4), please tell me if it's working for you.

**Reply by antoinechamot** [reply]
It is working, thanks for the great reactivity !
